pkg/hs/king/app/Main.hs

```diff
@@ -434,17 +436,30 @@ newShip CLI.New{..} opts
     runTryBootFromPill pill name ship bootEvent = do
       let pierConfig = toPierConfig (pierPath name) opts
       let networkConfig = toNetworkConfig opts
-      io $ runPierApp pierConfig networkConfig $
+      io $ runPierApp pierConfig networkConfig True $
```

Comment:
Should booting from a pill always trigger daemon mode? How does this interact with oExit?

Reply:
Right now, booting from a pill *always* exits instead of starting the ship.

I did this to hack around a problem with that flow. It was not committing something correctly, maybe to the event log? I don't remember exactly. Anyways, it was entering a bad state and other people were just starting to play with it, so I killed it.